fuel/app/classes/materia/widget/asset/manager.php

```diff
@@ -121,6 +123,13 @@ static public function delete_asset($id)
 		if (Widget_Asset_Manager::can_asset_be_deleted($id))
 		{
 			$asset = Widget_Asset::fetch_by_id($id);
+			// Uncomment to delete from S3
```

Comment:
We can probably leave this in, but all the asset manager deletion stuff appears to be vestigial - it's not actually used anywhere, as best I can tell. Simply removing the row from the db is safer, and the one place that actually "deletes" assets (in the `api/asset` controller) doesn't touch this code at all.